openstack_controller/datadog_checks/openstack_controller/api/api_rest.py

```diff
@@ -109,8 +109,8 @@ def authorize_project(self, project_id):
         self._current_project_id = project_id
 
     def _authorize_data(self, data):
-        self.log.debug("creating auth token")
-        response = self.http.post('{}/v3/auth/tokens'.format(self.config.keystone_server_url), json=data)
+        endpoint = '{}/v3/auth/tokens'.format(self.config.keystone_server_url)
```

Comment:
nit- this can be in another PR, but we could use f strings for everything in this file 
```suggestion
        endpoint = f'{self.config.keystone_server_url}/v3/auth/tokens'
```

Reply:
+1: I prefer f strings as well

Reply:
Question from a non-python expert: What is the benefit of one over the other beyond personal preferences? Do we have any established standards in integrations-core?

Reply:
After a bit of research:
f-strings provide several benefits over the .format() method:
- Readability and Conciseness: f-strings are more concise and easier to read than the .format() method. With f-strings, you directly embed variables and expressions inside the string, making the code more readable and reducing the number of function calls.
- Performance: f-strings are faster than the .format() method. Since f-strings are evaluated at runtime, they are more efficient than the .format() method, which involves method calls and parsing.
- Better Debugging: With f-strings, you can directly see the values of variables and expressions inside the string, which can be helpful for debugging. In contrast, with the .format() method, you may need to count placeholders and arguments to ensure they match correctly.
- Expression Evaluation: f-strings allow you to embed expressions directly inside the string, whereas the .format() method requires you to pass variables to the .format() function. This makes f-strings more flexible for complex expressions.
- No Method Call Overhead: f-strings are direct string literals with placeholders for variable values, while the .format() method involves calling a method on the string object. This method call overhead is eliminated with f-strings.

So yes, I'll do that change. Thanks for bringing it up!

Reply:
No problem! There are a lot of calls to the `format` method in this file, can we make a note to change this in the future?